float32 w_human
float32 h_human
float32 row_min_box
float32 row_max_box
float32 col_min_box
float32 col_max_box
float32 depth_human
